### PR TITLE
add training slide

### PR DESCRIPTION
**Pull request by YuikoTakada**

### CODE REVIEW

File: oss-training-2019-2.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# ��Q��׋��� Github�n���Y�I��
+
+
+---
```
Comment (shu-mutou) [inline oss-training-2019-2.md]: Add blank line after `---` paginatation.

Comment (YuikoTakada) [reply oss-training-2019-2.md]: done. sorry, I made one more commit by mistake :p

Comment (YuikoTakada) [reply oss-training-2019-2.md]: not shown images correctle :(

File: oss-training-2019-2.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+# アカウント作成
+
+---
+
+## アカウント作成①
+
+https://github.com/join にアクセスする。
+!(img/1-01.PNG)
```
Comment (shu-mutou) [inline oss-training-2019-2.md]: add `[]` between `!` and `(` like follow.
``

**Comment by shu-mutou** [comment]
Also, I fixed some issue, so please rebase to lates commit.

**Comment by shu-mutou** [comment]
Also, create folder for this contents and move all files into it.

File: oss-training-2019-2.md (hunk 2 above)
Comment (YuikoTakada) [reply oss-training-2019-2.md]: done, thanks

**Comment by YuikoTakada** [comment]
can I check this with slide mode?

**Comment by shu-mutou** [comment]
You can check it by accessing like follow:
https://shu-mutou.github.io/slideshow.html?md=https://raw.githubusercontent.com/YuikoTakada/shu-mutou.github.io/add_training_slide/oss-training-2019-2/oss-training-2019-2.md&title=Slideshow&theme=beige

You need to fix image url to full path due to spec of reveal.js.
e.g. `img/1-01.PNG` to `/img/oss-training-2019-2/1-01.PNG`

Although, images would not be shown until merging and published.

**Comment by shu-mutou** [comment]
To check all of your contents before merging, you need to deploy them into your http server.

File: oss-training-2019-2/oss-training-2019-2.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+https://github.com/join にアクセスする。
+![](img/1-01.PNG)
+
+---
+
+## アカウント作成②
+
+以下の情報を入力し、「Select a plan」ボタンをクリックする。
+- Username: 任意のユーザ名
+- Email address: 会社のメールアドレス
```
Comment (shu-mutou) [inline oss-training-2019-2/oss-training-2019-2.md]: 「会社の」はやめて、シンプルに「メールアドレス」にしましょう。

File: oss-training-2019-2/oss-training-2019-2.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+# 公開鍵の登録
+
+---
+
+## 公開鍵の登録①
+
+gitコマンドを使える環境で、以下のコマンドを実行してSSH Keyを作成する。
+
+```
+$ ssh-keygen -t rsa -C "<「アカウント作成①」で入力したメールアドレス>"`
```
Comment (atoato88) [inline oss-training-2019-2/oss-training-2019-2.md]: Is the last charactor `` ` `` a typo?

File: oss-training-2019-2/oss-training-2019-2.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    ProxyCommand nc --proxy-type http --proxy proxy.example.com:8080 %h %p
+```
+
+---
+
+## コード公開②
+
+以下のコマンドを実行して、作成したリポジトリをcloneする。
+```
+$ git clone git@github.com:<ユーザ名>/<リポジトリ名>.git`
```
Comment (atoato88) [inline oss-training-2019-2/oss-training-2019-2.md]: ditto.

**Comment by YuikoTakada** [comment]
- [x] コード公開① 
-> appendixに移動
SSHじゃなくてhttps認証の方法を書く
- [x] appendix2 
2要素認証
githubのリンクを貼っておく
- [x] PR作成⑤
git fetch upstream が抜けてる
- [x] PR作成⑫
×Submit new issue
〇Submit new pull request
- [x] 画像ファイルパス変更

**Pull request merged**